bokehjs/test/integration/regressions.ts

```diff
@@ -3744,4 +3744,26 @@ describe("Bug", () => {
       await display(p, [p.width! + left + 50, p.height! + top + 50])
     })
   })
+
+  describe("in issue #13692", () => {
+    describe("doesn't scale webgl antialising by pixel ratio", () => {
+      function plot(output_backend: OutputBackend) {
+        const p = fig([150, 150], {output_backend, title: output_backend})
+        p.line({x: [0, 1], y: [0, 1], line_width: 10})
+        return p
+      }
+
+      it.dpr(1)("with devicePixelRatio == 1", async () => {
+        await display(row([plot("canvas"), plot("svg"), plot("webgl")]))
+      })
+
+      it.dpr(2)("with devicePixelRatio == 2", async () => {
+        await display(row([plot("canvas"), plot("svg"), plot("webgl")]))
+      })
+
+      it.dpr(3)("with devicePixelRatio == 3", async () => {
```

Comment:
I'm not sure if just using `.dpr(N)` is enough to produce a meaningful test. I think this needs scaling as well, but we don't have an API exposed for this currently, though it's possible at CDP (Chrome Devtools Protocol) level.

Reply:
Yes, really we should also be scaling the canvas size along with the pixel ratio. But it is meaningful as it is, here is the dpr=3 test image before and after this PR:

![before](https://github.com/bokeh/bokeh/assets/580326/bcd8da7b-cbdf-4dfd-9148-d67583e74fc2)
![after](https://github.com/bokeh/bokeh/assets/580326/60e2ccc0-4c9c-4614-acf5-e87ce1da01cc)

The WebGL image is "too blurry" before this change, but one might have to enlarge the images to convince oneself.

Reply:
I'd happily remove the new test and save the disk space. But if we keep it we can at least identify when it changes and it will lead us back here where there are instructions for a manual test.